lang/cs/Org.Apache.REEF.Common/Org.Apache.REEF.Common.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Some of the *.csproj files had a hidden unicode character that were deleted when I made the change (I used the nuget package manager).